<ui version="4.0" >
 <class>StatisticsDlg</class>
 <widget class="QDialog" name="StatisticsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>393</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Statistics</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="sched_tab" >
      <attribute name="title" >
       <string>Schedule</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <layout class="QFormLayout" name="formLayout" >
         <property name="fieldGrowthPolicy" >
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Show stats for: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="comboBox" >
           <item>
            <property name="text" >
             <string>Next week</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Next month</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Next year</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>All-time</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget" >
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="next_week" />
         <widget class="QWidget" name="next_month" />
         <widget class="QWidget" name="next_year" />
         <widget class="QWidget" name="all_time" />
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="grades_tab" >
      <attribute name="title" >
       <string>Grades</string>
      </attribute>
      <layout class="QVBoxLayout" name="grades_layout" />
     </widget>
     <widget class="QWidget" name="categories_tab" >
      <attribute name="title" >
       <string>Categories</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" >
          <string>Coming soon! :P</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>StatisticsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>406</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel" >
     <x>238</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
